bin/plotDrugsAcrossCells.R

```diff
@@ -26,6 +26,81 @@ plotDrugByCellAndTumor<-function(compoundName,tumorType){
       ggsave(fname)
     }}
   return(fname)
- # ggplot(red.tab)+geom_violin(aes(x=symptom_name,y=response,col=organism_name))+facet_grid(.~response_type)+ggtitle(paste(compoundName,'response across cells'))+theme(axis.text.x = element_text(angle = 45, hjust = 1))
+  # ggplot(red.tab)+geom_violin(aes(x=symptom_name,y=response,col=organism_name))+facet_grid(.~response_type)+ggtitle(paste(compoundName,'response across cells'))+theme(axis.text.x = element_text(angle = 45, hjust = 1))
   
 }
+
+screening_data_syn_id <- "syn17100888"
+
+screening_data_with_id <- synGet(screening_data_syn_id)$path %>% 
```

Comment:
Is there some advantage to having this load by default? Or should we load it by a function?

Reply:
this is what happens in the script above (`synId`) so i was just following that example. 

unclear what is the better option, but if the drug screening data aren't formatted exactly as these are nothing will work.

Reply:
Yeah, I realize i was being inconsistent. let's keep it like this for now.